producerjson/README.md

```diff
@@ -0,0 +1,46 @@
+# producerjson
+
+Producer JSON smart contract for EOS.
+
+# Usage
+
+### View the table
+
+```
+cleos get table producerjson producerjson producerjson
+```
+
+### Add to the table
+
+```
+cleos push action producerjson set '{"owner":"your_account", "json": "your_json"}' -p your_account@active
+```
+
+**Example**:
+```
+cleos push action producerjson set '{"owner":"teamgreymass", "json": "'`printf %q $(cat bp.json | tr -d "\r")`'"}' -p teamgreymass@active
+```
+
+### Remove from the table
+
+```
+cleos push action producerjson del '{"owner":"your_account"}' -p your_account@active
+```
+
+
+---
+
+# How to build
+```
+./build.sh
+```
+or
+```
+eosiocpp -g producerjson.abi producerjson.cpp && eosiocpp -o producerjson.wast producerjson.cpp
```

Comment:
What compile are we using here?

https://developers.eos.io/welcome/latest/getting-started-guide/hello-world/#procedure-to-compile-and-deploy